packs/blog-bestiary/foolish-housekeeping-and-other-articles/rt5rrmn.json

```diff
@@ -0,0 +1,324 @@
+                    ]
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "pmZw64m62ArdHicK",
+            "img": "systems/pf2e/icons/actions/TwoActions.webp",
+            "name": "Yidsfh",
+            "sort": 600000,
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "category": "offensive",
+                "description": {
+                    "value": "<p>The rt5rrmn Strides and then makes two g7ujik Strikes at a single target. Both count toward its multiple attack penalty, but the penalty increases only after both attacks are made. The rt5rrmn must be able to move at least half its Speed to yidsfh.</p>"
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [],
+                "slug": null,
+                "traits": {
+                    "value": []
+                }
+            },
+            "type": "action"
+        }
+    ],
+    "name": "Rt5rrmn",
+    "system": {
+        "abilities": {
+            "cha": {
+                "mod": 6
+            },
+            "con": {
+                "mod": 4
+            },
+            "dex": {
+                "mod": 3
+            },
+            "int": {
+                "mod": 1
+            },
+            "str": {
+                "mod": 5
+            },
+            "wis": {
+                "mod": 1
+            }
+        },
+        "attributes": {
+            "ac": {
+                "details": "",
+                "value": 36
+            },
+            "allSaves": {
+                "value": ""
+            },
+            "hp": {
+                "details": "",
+                "max": 172,
+                "temp": 0,
+                "value": 172
+            },
+            "speed": {
+                "otherSpeeds": [],
+                "value": 35
+            }
+        },
+        "details": {
+            "blurb": "",
+            "languages": {
+                "details": "(can’t speak any language)",
+                "value": [
+                    "common"
+                ]
+            },
+            "level": {
+                "value": 8
+            },
+            "privateNotes": "<p><em>Note - Paizo did not provide an attack modifier for any of the attacks. We used the moderate Strike Attack Bonus for the creature level (GM Core pg. 120).</em></p>",
+            "publicNotes": "<p>The rt5rrmn is a catlike creature that loves to purr at those they encounter, convincing others to pamper them instead of fight. It would rather be friends than enemies, but if spooked it will viciously attack whoever is in its sights. If they aren't able to make friends, the rt5rrmn will</p>",
```

Comment:
```suggestion
            "publicNotes": "<p>The rt5rrmn is a catlike creature that loves to purr at those they encounter, convincing others to pamper them instead of fight. It would rather be friends than enemies, but if spooked it will viciously attack whoever is in its sights. If they aren't able to make friends, the rt5rrmn will quickly grow defensive, and leap to attack. </p>",
```